src/components/DropDownMenu.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import Checked from '../images/iconfinder_done-01_186405.svg'
+export const DropDownMenu = ({ isOpen, setFormat, format }) => {
+  const formatOptions = [{
+    id: 1,
+    type: 'Plain Text'
+  }, {
+    id: 2,
+    type: 'HTML'
+  }]
+  return (
+    <div>
+      {
+        isOpen && <div style={{
+          display: 'flex',
+          flexDirection: 'column',
+          justifyContent: 'space-evenly',
+          border: '1px solid white',
+          width: '108px',
+          height: '65px'
+        }}>
+          { isOpen &&
+          formatOptions.map(el => (
+            <div
+              style={{ cursor: 'pointer' }}
+              key={el.id}
```

Comment:
You can use the `type` or the map index (second argument to the mapping function) as the key instead of explicitly defining them.